chringo/lib/pages/bingo_page.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:chringo/records/wordbanks_record.dart';
```

Comment:
This file is named bingo_page, which is perfect. However, I'm not sure what's the best way to library-itize our code at this point. For sure the first thing that needs to happen is that we move our dart source code into `lib/src/...` instead of directly in the lib. then we could maybe think about defining files in `lib/` which export either specific files or specific libraries. For now, let's just move the new source code into `lib/src/` and then work from there. Or we can do that in a followup 🤷‍♂

Reply:
Move all code we have into `lib/src/...`? I.e. `main.dart` and the directories containing other files?